### DISCUSSION TITLE
Next release projection?

### DISCUSSION THREAD

**Discussion by dreed47** [pawelsalawa/sqlitestudio#4446] [General]
Many bugs have been fixed since last release and Im aware of your policy on not producing betas but the app is difficult to build from source leaving people no option but to use the app with the bugs or find an alternative.  Specifically Im looking for these fixes https://github.com/pawelsalawa/sqlitestudio/issues/4207 

Any ideas on release date or should I figure something else out.  Thanks for all your hard work.

**Comment by pawelsalawa** [comment]
You can download binaries from nightly builds: https://github.com/pawelsalawa/sqlitestudio/actions

**Reply by dreed47** [reply]
thanks, ill give that a shot
